testConnectivity.py

```diff
@@ -236,7 +239,17 @@ def testConnectivity(args):
             pass
 
         vfatBoard.parentOH.parentAMC.writeRegister("GEM_AMC.TTC.GENERATOR.ENABLE",0x0)
-        
+
+        print("Checking trigger link status:")
+        for trial in range(0,args.maxIter ):
+            testLinks = vfatBoard.parentOH.parentAMC.getTriggerLinkStatus(printSummary=True, checkCSCTrigLink=False, ohMask=args.ohMask)
+	    if (testLinks < 1):
+                printGreen("Trigger link to OHs in mask:{0} is good".format(hex(args.ohMask)))
+                break
+            else:
+                printRed("Trigger link to OHs in mask:{0} failed".format(hex(args.ohMask)))
+                vfatBoard.parentOH.parentAMC.writeRegister("GEM_AMC.TRIGGER.CTRL.CNT_RESET",0x1)
```

Comment:
Ah I think this is the wrong reset (I probably told you the wrong register, my bad).  Here you should be using:

`GEM_AMC.TRIGGER.CTRL.MODULE_RESET`

This will reset the module, while above will just reset the counters (and cause them to roll up again if there's an issue).